dotcom-rendering/src/web/components/Card/Card.tsx

```diff
@@ -348,12 +353,12 @@ export const Card = ({
 								/>
 							) : undefined}
 						</HeadlineWrapper>
-						{avatar && (
+						{imageType === 'avatar' && avatarUrl && (
```

Comment:
Using the helper method, you wount no longer have to check for the truthiness of `avatarUrl`, but rather can assume that `image` is always string if `imageType` is defined.

Reply:
I added the extra check here to keep typescript happy. We know from reading the code that if imageType === avatar then avatarUrl can't be undefined but ts doesn't

Reply:
Hence my proposal to create a helper that can help discriminate.